test/controllers/api/v2/systems_controller_test.rb

```diff
@@ -244,5 +247,21 @@ def test_product_content
       assert_response :success
       assert_template 'api/v2/systems/product_content'
     end
+
+    def test_add_subscriptions
```

Comment:
could you rename this test `test_add_subscriptions_protected` ? APJ from me pending this line change. Great job!